website/pages/docs/upgrading/upgrade-to-1.4.0.mdx

```diff
@@ -7,4 +7,24 @@ description: |-
   for Vault 1.4.0. Please read it carefully.
 ---
 
-There are no version-specific upgrade procedures associated with version 1.4.0.
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.3.4 compared to 1.4.0. Please read it carefully.
+
+## The AWS Auth Engine
+
+Users of the AWS Auth Engine should be cautious with this upgrade, because in
+1.4.0 we began adding metadata to tokens issued with this method. While the
+metadata does help with tying tokens to a particular person or machine, it
+also can also take a performance toll.
+
+The specific fields added for `iam` tokens were `client_arn`, `canonical_arn`,
+`client_user_id`, `auth_type`, `inferred_entity_type`, `inferred_entity_id`,
+`inferred_aws_region`, and `account_id`. The specific fields added for `ec2`
+logins were `instance_id`, `region`, `account_id`, and `ami_id`.
+
+Companies who may experience a high rate of change in these fields for a role
```

Comment:
Companies -> users, maybe? Companies seems like a word we don't use often in our docs, and you use 'users' earlier in this section.

Do we have an example of the type of situation where you might see a high rate of change for these fields? I can't visualize this, but I'm also probably not really the intended audience.

Reply:
Hi! Thanks for looking at this.

I tried elaborating more about who would see a high rate of change [here](https://github.com/hashicorp/vault/pull/8731/files#diff-3772dd3d3374cb57b6e5e8656299772aR8) and down. What do you think?